test/test_operations.py

```diff
@@ -1649,6 +1649,40 @@ def test_cached_addcdiv(self):
     xm.mark_step()
     self.assertEqual(met.metric_data("TransferToServerTime")[0], 4)
 
+  def test_print_executation(self):
+    xla_device = xm.xla_device()
+    xm.mark_step()
+    met.clear_all()
+
+    # case 1 mark_step
+    t1 = torch.randn(1, 4, device=xla_device)
+    xm.mark_step()
+    xm.wait_device_ops()
+    self.assertEqual(met.metric_data('ExecuteTime')[0], 1)
+    for _ in range(3):
+      print(t1)
+    self.assertEqual(met.metric_data('ExecuteTime')[0], 1)
+    self.assertIn('xla::device_data',
```

Comment:
I wonder what this`self.assertIn('xla::device_data'` is testing for.

Reply:
make sure t1 is a `device_data` after print, this mean execution has materialize the tensor.